README.md

```diff
@@ -44,7 +44,9 @@ development environment for the other TDR services.
   - Set the Tasks parameter to `run`
   - Configure the environment variables:
     - AUTH_SECRET=\<the secret for the Keycloak client that you copied above\>
+    - READ_AUTH_SECRET=\<follow the steps above for obtaining `AUTH_SECRET`, but using parameter key `/intg/keycloak/user_read_client/secret`\>
     - NOTIFICATION_SNS_TOPIC_ARN=\<the arn for sns topic for slack notifications you obtained above>
```

Comment:
Thanks for updating. Can you also add a line for also configuring
`AUTH_URL=https://auth.tdr-integration.nationalarchives.gov.uk`